homeassistant/components/updater/__init__.py

```diff
@@ -1,8 +1,7 @@
 """Support to check for available updates."""
+# pylint: disable=import-error
```

Comment:
Let's move this behind the import where it was meant for (line 5)

Reply:
I think it belongs at `from distro import linux_distribution`.

Now the test passes locally.